APIProject/EventAPI/Controllers/EventController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Karma.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace EventAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EventController : ControllerBase
+    {
+        [HttpGet]
+        public string GetEvents(string address)
```

Comment:
You can use ActionResult<List<CharityEvents>> (could return 404 if no events would be found)

Reply:
Added special output when there are no search results